src/Models/HydrostaticFreeSurfaceModels/mg_implicit_free_surface_solver.jl

```diff
@@ -78,72 +97,19 @@ function MGImplicitFreeSurfaceSolver(grid::AbstractGrid,
     settings[:maxiter] = get(settings, :maxiter, grid.Nx * grid.Ny)
     settings[:reltol] = get(settings, :reltol, min(1e-7, 10 * sqrt(eps(eltype(grid)))))
```

Comment:
is this `10 * ` left over from debugging?

Reply:
Yes I'll delete it

Reply:
if we just leave `:reltol, sqrt(eps(eltype(grid))))` will it error?